samples/subsys/usb/uvc/sample.yaml

```diff
@@ -0,0 +1,13 @@
+sample:
+  name: USB Video sample
+tests:
+  sample.subsys.usb.uvc:
+    depends_on:
+      - usbd
+      - video
+    tags: usb video
+    platform_allow:
+      - nrf52840dongle
+      - mini_stm32h743
+      - rpi_pico
+    harness: TBD
```

Comment:
TBD for "To Be Done", TODO would have been more conventional...

I need to add a proper test harness for this sample.
This might require the user to quickly react to start the stream before a test timeout is reached...
I can take inspiration on the other video tests maybe.

Reply:
I mean this should not be written `harness: TBD` as TBD is not a valid harness test. It should be implemented or even if it's not, this should be removed. The default test timeout is 10s and it is changeable (I couldn't remember the command to change the timeout though)

Reply:
I removed it in the meantime that a test harness is setup.